protostar/start.py

```diff
@@ -1,23 +1,28 @@
 import asyncio
 from pathlib import Path
 
+from protostar.cli import ArgumentParserFacade, ArgumentValueFromConfigProvider
 from protostar.cli.cli_app import CLIApp
-from protostar.protostar_cli import ProtostarCLI
-from protostar.cli import ArgumentValueFromConfigProvider
-from protostar.cli import ArgumentParserFacade
+from protostar.protostar_cli import ConfigurationProfileCLI, ProtostarCLI
 
 
 def main(script_root: Path):
     protostar_cli = ProtostarCLI.create(script_root)
 
     parser = ArgumentParserFacade(
         protostar_cli,
-        default_value_provider=ArgumentValueFromConfigProvider(protostar_cli.project),
+        default_value_provider=ArgumentValueFromConfigProvider(
+            protostar_cli.project,
+            configuration_profile_name=ArgumentParserFacade(
+                ConfigurationProfileCLI(), disable_help=True
+            )
+            .parse(ignore_unrecognized=True)
+            .profile,
+        ),
```

Comment:
```suggestion
        current_profile = ArgumentParserFacade(
                ConfigurationProfileCLI(), disable_help=True
            )
            .parse(ignore_unrecognized=True)
            .profile,

        default_value_provider=ArgumentValueFromConfigProvider(
            protostar_cli.project,
            configuration_profile_name=current_profile
        ),
```
Nitpick but I would prefer it this way, it's a bit messy right now